pyproject.toml

```diff
@@ -84,8 +84,8 @@ include = [
 
 [tool.rapids-build-backend]
 build-backend = "setuptools.build_meta"
-commit-files = ["_nx_cugraph/GIT_COMMIT"]
-dependencies-file = "../../dependencies.yaml"
+commit-files = ["nx_cugraph/GIT_COMMIT"]
+dependencies-file = "dependencies.yaml"
```

Comment:
Wheel tests were failing like this:

> ERROR: Could not find a version that satisfies the requirement pylibcugraph==24.12.*,>=0.0.0a0 (from nx-cugraph-cu12) (from versions: none) 

([build link](https://github.com/rapidsai/nx-cugraph/actions/runs/11735478731/job/32693366701?pr=15#step:9:47))

That's because the wheel **builds** were not correctly setting metadata about what packages `nx-cugraph` depends on. `rapids-build-backend` was warning but not failing here:

>   /pyenv/versions/3.12.7/lib/python3.12/contextlib.py:137: UserWarning: File not found: '../../dependencies.yaml'. If you want rapids-build-backend to consider dependencies from a dependencies file, supply an existing file via config setting 'dependencies-file'.

([build link](https://github.com/rapidsai/nx-cugraph/actions/runs/11735478731/job/32693252927?pr=15#step:13:112))

For background, it was an intentional choice to make that case a warning instead of an error: https://github.com/rapidsai/rapids-build-backend/pull/33#discussion_r1602081552